com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [2.2.0-exp.3] - 2021-01-22
 ### Changes
```

Comment:
`Changes` => `Changed`
I know we didn't officially state on the changelog section names, but we now tend to uniformize with these (in this order):
- Added
- Changed
- Fixed